sandbox-templates/remark-with-parcel/src/index.js

```diff
@@ -0,0 +1,25 @@
+// ⚠️ Important! Please make sure the dependencies are up to date.
+// You can refresh them in the Dependencies section (left-bottom on CodeSandbox)
```

Comment:
with stackblitz also being linked should this comment be updated?
suggestions on new copy?

Reply:
I dunno where stackblitz is 🤔

```
// On CodeSandbox, them in the Dependencies section (left-bottom)
// On StackBlitz, ...
```

^-- something like this?